templates/download/desktop/index.html

```diff
@@ -188,19 +194,43 @@ <h2>Ubuntu {{ releases.latest.full_version }}</h2>
             The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu 25.04 comes with nine months of security and maintenance updates, until January 2026.
           </p>
           <hr class="p-rule--muted" />
-          <p class="u-responsive-realign">
-            <a class="p-button--positive"
-               href="/download/desktop/thank-you?version={{ releases.latest.full_version }}&amp;architecture=amd64"
-               onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Desktop', 'eventLabel' : '{{ releases.latest.short_version }}', 'eventValue' : undefined });">Download
-              {{ releases.latest.full_version }}&nbsp;(AMD64)
-            </a>
-            {% if releases.latest.iso_download_size %}
-              <span class="u-text--muted">{{ releases.latest.iso_download_size }}</span>
-            {% endif %}
-            <script>
-              performance.mark("Download (Desktop) button rendered")
-            </script>
-          </p>
+          <div class="u-responsive-realign">
```

Comment:
Same comment as above, we can also remove the `u-responsive-realign` as it was intended for a slightly different implementation